docker-compose.yml

```
@@ -113,7 +113,10 @@ services:
       - st2api
     networks:
       - private
+    environment:
+      - SSH_AUTH_SOCK=/ssh-agent
     volumes:
+      - /run/host-services/ssh-auth.sock:/ssh-agent
```

Can you do something more generic?
On my linux box I have something like:
`SSH_AUTH_SOCK=/tmp/ssh-XXXXXXs4KUHi/agent.1234`

So, is there a way to avoid hard-coding this path?

---

I don't know docker-compose very well, but would something like this work?
```suggestion
      - ${SSH_AUTH_SOCK:-/run/host-services/ssh-auth.sock}:/ssh-agent
```

---

+1.

Another thought is how to make it optional?

I'm thinking that majority of the users might not need these volume mounts by default
So we avoid sharing the ssh auth socket with st2 unless configured.